io/src/test/scala/fs2/io/file/FilesSuite.scala

```diff
@@ -263,7 +263,7 @@ class FilesSuite extends Fs2Suite with BaseFileSuite {
               files.exists(tempDir.resolve(file.getFileName))
             }
         }
-        .map(it => assert(it))
+        .map(existsInTemp => assertEquals(existsInTemp, true))
```

Comment:
sorry, what I meant is that you can just call `.assertEquals(true)` instead of doing the `.map` at all. Sorry for the confusion

Reply:
Ah right, that _does_ make more sense than how I interpreted it at first 😅 serves me right for only glancing over munit's docs. 

Will get it amended now.

Reply:
Thanks. It's the munit cats-effect integration that Michael added very recently which lets you do that

Reply:
That's done now